configs/eslint-config-ts/typescript.js

```diff
@@ -1,10 +1,27 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
+  root: true,
   extends: ['plugin:import/typescript', './noimport.js'],
   plugins: ['import'],
   settings: {
     'import/resolver': {
       typescript: {},
     },
   },
+  overrides: [
+    {
+      files: ['*.test-d.ts*', '*.test-d.ts*'],
+      rules: { 'import/no-unresolved': ['error', { ignore: ['tsd'] }] },
+    },
+  ],
```

Comment:
Move this section from `eslint-config-js/javascript.js` to `eslint-config-ts/typescript.js`. because I don't want to make `eslint-config-js/javascript.js` care `tsd` for testing TypeScript declaration files.